committee-steering/governance/annual-reports.md

```diff
@@ -0,0 +1,125 @@
+# Kubernetes Community Group Annual Reports
+
+This document outlines the bootstrap process for SIG, WG, and UG Chairs and Tech
+Leads and Committees to establish a community group health check with Steering.
+All policy updates will be in their respective [SIG], [WG], and [UG]
+ governance docs as well as the general [governance] guidance.
+
+## Goals
+- Paint a complete project health picture for all of our community groups
+- Create a feedback loop between Chairs, the community groups, and the Steering
+Committee to move the project forward  
+- Encourage dialogue about the wellbeing of the projects contributors and offer
+suggested guidance and coaching
+- Promote healthy, active, engaged community groups  
+- Understand and have context before issues arise and celebrate wins where they  
+should be highlighted  
+- Help reshape project priorities at a high level
+
+## Reporting Process  
+
+Chairs are responsible for compiling a yearly public report but may be
+completed with the help of members of that group. Chairs are responsible for
+ensuring the reports are complete, accurate, and submitted to the Steering
+Committee.
+
+### 2020:
+Due to playing catch up with 40 community groups, we will have a slightly
+modified timeline bootstrap process this year and then will follow the process
+outlined in 2021. This means that Chairs will be reporting twice in n months and
+not a year. A yearly cadence will be followed post 2021.
+
+TODO outline time line for 2020  
+
+### 2021 and on:
+1. Chairs create one report and send it to steering-private@kubernetes.io for
+review; reports are due at the end of February every year and can pick a
+collaboration doc tool of their choice (google docs, hackmd, etc)
+2. Steering Committee will assign one member to each community group before the
+reports are submitted so there is an established point of contact for any help or
+guidance needed
+3. The March SIG Chairs and Tech Leads meeting will be used as follow up and
+report retrospectives
```

Comment:
It's unclear to me if the March SIG Chairs/TL call will be used to discuss general followup/retro themes we noticed across _all_ community groups or will we discuss a particular SIG/few SIGs in that call?